dns/route53/dns_integration_test.go

```diff
@@ -0,0 +1,65 @@
+// +build integration
+
+package route53
+
+import (
+	"testing"
+
+	"fmt"
+	"net"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+)
+
+// NOTE: this test requires valid AWS credentials being available
+// DOCS: https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials
```

Comment:
Please refer to https://blog.golang.org/godoc and start the comment with the name of the thing you're documenting.

Reply:
`_test.go` wont how up in documentation. So I guess this has no impact.

Reply:
I'm fine not having comments for unit tests, as in https://github.com/wireapp/srv-announcer/pull/5/files#r492506981, but all comments we have should follow the same style guide, which asks for them to start it with the name of the thing your documenting:

> The convention is simple: to document a type, variable, constant, function, or even a package, write a regular comment directly preceding its declaration, with no intervening blank line. Godoc will then present that comment as text alongside the item it documents. […]

In our case, this could mean, starting this with:

```
TestIntegrationAddAndRemoveATarget adds and removes a target from a zone already existing in Route53.
It ensures the record set is removed as soon if it would be empty, as Route 53 doesn't allow empty resource record sets.
NOTE: […]
```